ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1185,6 +1188,10 @@ objects:
             echo "Unsupported cluster type '${CLUSTER_TYPE}'"
             exit 1
         fi
+        openshift-install --dir=/tmp/artifacts/installer create manifests
+        rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-*.yaml
+        rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+        sed -i "s;mastersSchedulable: true;mastersSchedulable: false;g" /tmp/artifacts/installer/manifests/cluster-scheduler-02-config.yml
```

Comment:
Should this be up in the `elif [[ "${CLUSTER_TYPE}" == "vsphere" ]]; then` block like the others?

Reply:
@jstuever thanks for catching that.